src/language/githubReleaseInstaller.ts

```diff
@@ -15,18 +15,29 @@ import { DafnyInstaller, getPreferredVersion } from './dafnyInstallation';
 import { configuredVersionToNumeric } from '../ui/dafnyIntegration';
 const ArchiveFileName = 'dafny.zip';
 
-function getDafnyPlatformSuffix(): string {
+function getDafnyPlatformSuffix(version: string): string {
+  const post312 = version.includes('nightly') || configuredVersionToNumeric(version) >= configuredVersionToNumeric('3.13');
```

Comment:
The 'nightly' part is slightly wrong, since in the context of this method it could be an older nightly build. If that's not actually a concern because of what this function is called on, we should add a comment.

Reply:
Since it always fetches the most recent nightly, I don't think it can fail. I'll add a comment about that.